src/formik.tsx

```diff
@@ -91,14 +95,20 @@ export interface FormikState<V> {
 export interface FormikActions<P, V> {
   /* Manually set top level error */
   setError: (e: any) => void;
-  /* Manually set Errors */
+  /* Manually set errors object */
   setErrors: (errors: FormikErrors) => void;
   /* Manually set isSubmitting */
   setSubmitting: (isSubmitting: boolean) => void;
-  /* Manually set touched fields */
+  /* Manually set touched object */
   setTouched: (touched: FormikTouched) => void;
-  /* Manually set values  */
+  /* Manually set values object  */
   setValues: (values: V) => void;
+  /* Set value of form field directly */
+  setFieldValue: (field: string, value: any) => void;
```

Comment:
Weren't we going to add a `touched?: boolean` to the end of this signature? I think I like it better without it actually on the basis of single responsibility principle, but we should acknowledge that the consequence is that a number of components will have to make calls to both `setFieldValue` and `setFieldTouched`, meaning double the number of state updates, which is sort of annoying.